typed-racket-lib/typed-racket/utils/inspector.rkt

```diff
@@ -0,0 +1,10 @@
+#lang racket/base
+
+(provide old-inspector new-inspector)
+
+;; set up a new inspector that typed racket structs will use,
+;; and make the new inspector a sub-inspector of the old one.
+(define old-inspector (current-inspector))
+(define new-inspector (make-inspector old-inspector))
+(current-inspector new-inspector)
```

Comment:
Yes, relative phase 0 for `core.rkt` is phase 1 for the file using `#lang typed/racket`.